front/src/Components/Chat/ChatForum.svelte

```diff
@@ -0,0 +1,33 @@
+<script>
```

Comment:
@cesar-cardinale this file is used in `front/src` or already moved in `front/chat/src`?